lldb/tools/driver/CMakeLists.txt

```diff
@@ -11,13 +11,23 @@ if(APPLE)
   set(CMAKE_EXE_LINKER_FLAGS "${CMAKE_EXE_LINKER_FLAGS} -Wl,-sectcreate,__TEXT,__info_plist,${CMAKE_CURRENT_BINARY_DIR}/lldb-Info.plist")
 endif()
 
+## BEGIN SWIFT
+if (NOT CMAKE_SYSTEM_NAME MATCHES "Darwin|Windows")
+  set(option_install_rpath INSTALL_RPATH "$ORIGIN/../lib")
```

Comment:
I checked and lldb is the only tool that still needs this relative rpath that I removed by setting `LLDB_NO_INSTALL_DEFAULT_RPATH` above, so add that alone back.

Reply:
Also, I didn't bother adding a `BUILD_RPATH` too, since the Swift build doesn't test these LLVM binaries, but can add that too if wanted.

Reply:
Went back and added `BUILD_RPATH` too, as these lldb binaries alone _are_ tested on the Swift CI.